manifests/client.pp

```diff
@@ -44,12 +44,35 @@
           content => template("${module_name}/sensu-client.erb"),
         }
 
-        exec { 'install-sensu-client':
```

Comment:
Why does the exec go away entirely? I would assume it would still be present if you are not using the new dsc code.